atum/src/main/scala/za/co/absa/atum/core/Atum.scala

```diff
@@ -65,12 +65,23 @@ object Atum {
 
   /**
     * Returns the current control measures object containing all the checkpoints up to the current point.
+    *
+    * @deprecated This method is deprecated. Please use getControlMeasure
     */
+  @Deprecated
   def getControMeasure: ControlMeasure = {
     preventNotInitialized()
```

Comment:
It's generally best practice to have the deprecated method call its own replacement to avoid duplication (in this case two lines in a fairly stable project, so not a big issue).

Reply:
Seems like the general best practice differs a lot. If you want to, go ahead and create a new PR. I am standing by my thought of not touching deprecated methods and always keeping them as they are.

Reply:
Like I said, not a big issue.

Reply:
Didn't know that. Interesting. Thanks!